src/azure-account.ts

```diff
@@ -40,19 +40,17 @@ const environmentLabels: Record<string, string> = {
 	[azurePPE]: localize('azure-account.azurePPE', 'Azure PPE'),
 };
 
-const logVerbose = false;
+const logVerbose: boolean = false;
 
-interface AzureAccountWriteable extends AzureAccount {
+interface IAzureAccountWriteable extends AzureAccount {
```

Comment:
Should we follow this rule of prefixing interfaces with an "I" given that eslint doesn't require this anymore? 
If we plan to, we should probably add this rule for eslint checks. 
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-interface-names-do-not-begin-with-an-i